pkgs/by-name/nb/nbfc-linux/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  callPackage,
+  lib,
+  stdenv,
+  fetchurl,
+  nixos,
+  testers,
+  hello,
+  fetchgit,
+}:
+stdenv.mkDerivation (finalAttrs: let
+  buildFlags = ["PREFIX=$(out)" "confdir=/etc"];
+  installFlags = ["PREFIX=$out" "confdir=$out/etc"];
+in rec {
+  inherit buildFlags;
+
+  pname = "nbfc-linux";
+  version = "0.1.15";
+
+  src = fetchgit {
+    url = "https://github.com/nbfc-linux/nbfc-linux.git";
+    rev = "c2406d2e64341e91d0a040ee2157e67dd02ccc79";
+    hash = "sha256-3+2qfW4Oo5/cLt5Ya7pHnh1t6hURYtaIEqM1CPS5R8E=";
+  };
```

Comment:
Please use `fetchFromGitHub` with `version = "v${finalAttrs.version}"`. This is more efficient than using `fetchgit` because it downloads just the version tarball instead of the whole repo.